src/state/slices/actionSlice/types.ts

```diff
@@ -29,8 +30,14 @@ export enum ActionStatus {
   Cancelled = 'Cancelled',
 }
 
+export enum ArbitrumBridgeType {
+  Deposit = 'Deposit',
+  Withdraw = 'Withdraw',
+}
```

Comment:
blocking: this is used for logic purposes only, and we already have `ArbitrumBridgeSpecificMetadata` we can expose, no need to reinvent the wheel (it's already part of the swap btw, so you just need to type the metadata properly, there's not even a need to re-do the `isDeposit` / `isWithdraw`)